src/test/java/basesyntax/reader/FileReaderImplTest.java

```diff
@@ -0,0 +1,47 @@
+package basesyntax.reader;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileReaderImplTest {
+    private FileReaderInterface fileReader;
+
+    @BeforeEach
+    void setUp() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test
+    void read_validFiles_Ok() throws IOException {
+        Path tempFile = Files.createTempFile("test", ".csv");

```

Comment:
Use the test/resources directory for the test files. Move the path to the constant fields